app/controllers/admin/software_updates_controller.rb

```diff
@@ -6,7 +6,7 @@ class SoftwareUpdatesController < BaseController
 
     def index
       authorize :software_update, :index?
-      @software_updates = SoftwareUpdate.by_version
+      @software_updates = SoftwareUpdate.by_version.filter(&:pending?)
```

Comment:
Not necessary for this PR ... but it may make sense at some point to do a query object sort of thing, or some other approach to the various "scope"-like class methods on `SoftwareUpdate`, so the logic can stay in the class and externally have interface more like AR class scopes.